src/main/java/seedu/address/logic/commands/quizcommands/AnswerCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands.quizcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.QuizModel;
+import seedu.address.model.flashcard.Answer;
+
+public class AnswerCommand extends Command<QuizModel> {
+
+    public static final Status STATUS = Status.ON_QUESTION;
+    private final String answer;
+
+    public AnswerCommand(String answer) {
+        this.answer = answer;
+    }
+
+    @Override
+    public CommandResult execute(QuizModel model) throws CommandException {
+        requireNonNull(model);
+
+        if (!model.hasStarted()) {
+            throw new CommandException(QuizCommand.MESSAGE_QUIZ_NEVER_STARTED);
+        }
+
+        if (!QuizCommand.getStatus().equals(STATUS)) {
+            throw new CommandException(QuizCommand.MESSAGE_UNAVAIL_ON_ANSWER);
+        }
+
+        model.saveAnswer(answer);
+        Answer answer = model.getAnswer();
+
+        QuizCommand.setStatus(Status.ON_ANSWER);
+        QuizCommand.updateCommandResult(answer.toString());
+
+        return new CommandResult(answer.toString());
```

Comment:
Maybe you can add a SUCCESS_MESSAGE for it e.g. `String.format("Your answer: %s, answer)`, I think that would be more user friendly.